src/appengine/server_flask.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""server_flask.py initialises the appengine server for ClusterFuzz."""
+from base import utils
+from config import local_config
+from flask import Flask
+from google.cloud import ndb
+from handlers import base_handler
+from handlers import jobs
+from metrics import logs
+from system import environment
+
+_is_chromium = utils.is_chromium()
```

Comment:
nit: move these down closer to where they're used.

Reply:
done